### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by ronag**
- [ ] At least two implementers are interested (and none opposed):
   * …
   * …
- [ ] [Tests](https://github.com/web-platform-tests/wpt) are written and can be reviewed and commented upon at:
   * …
- [ ] [Implementation bugs](https://github.com/whatwg/meta/blob/main/MAINTAINERS.md#handling-pull-requests) are filed:
   * Chrome: …
   * Firefox: …
   * Safari: …

(See [WHATWG Working Mode: Changes](https://whatwg.org/working-mode#changes) for more details.)

***
<a href="https://whatpr.org/fetch/1290.html" title="Last updated on Aug 31, 2021, 1:12 PM UTC (7e53dfe)">Preview</a> | <a href="https://whatpr.org/fetch/1290/613aad9...7e53dfe.html" title="Last updated on Aug 31, 2021, 1:12 PM UTC (7e53dfe)">Diff</a>

### CODE REVIEW

**Comment by annevk** [comment]
Hey @ronag, could you sign https://participate.whatwg.org/agreement please?

**Comment by ronag** [comment]
Signed

File: fetch.bs [hunk 1]
```diff
@@ -5993,7 +5993,7 @@ new Headers(meta2);
   <var>object</var>:
 
   <ol>
-   <li><p>If <var>header</var> does not contain exactly two items, then <a>throw</a> a
+   <li><p>If <var>header</var> is not a <a>sequence</a> which contains exactly two items, then <a>throw</a> a
```
Comment (annevk) [inline fetch.bs:5996]: I looked at this again to be sure and it doesn't seem to be needed. IDL wouldn't allow this if it were not a sequence. I would be okay with adding a step before this one that asserts it's a sequence though.

**Review by annevk — COMMENTED** [review]
Thanks. After some further consideration I suggest either changing or removing the second change as it's currently redundant with Web IDL requirements.

File: fetch.bs (hunk 1 above)
Comment (ronag) [inline fetch.bs:5996]:
```suggestion
   <li><p>If <var>header</var> does not contain exactly two items, then <a>throw</a> a
```

**Pull request merged**